### PR TITLE
Hanan/activate validators

### PR DESCRIPTION
**Pull request by HananINouman**
## Summary
Activate validators section. 

## Screenshots
Screenshot 2023-10-07 at 8 19 49 PM

ticket:
#266

### CODE REVIEW

File: docs/int/quickstart/advanced/quickstart-sdk.md [hunk 1]
```diff
@@ -88,21 +90,48 @@ Once the Obol-API returns a `configHash` string from the `createClusterDefinitio
...
 const clusterLock = await obol.getClusterLock(configHash);
 ```
 
-Reference lock files can be found [here](https://github.com/ObolNetwork/charon/tree/main/cluster/testdata). 
+Reference lock files can be found [here](https://github.com/ObolNetwork/charon/tree/main/cluster/testdata).
 
 ## Activate the DVs using the deposit contract
 
-This guide will in future cover activating the deposit data from a lock file.
+In order to activate cluster validator, the cluster operator can retrieve the validator associated deposit data and call deposit from the deposit contract.
```
Comment (OisinKyne) [inline docs/int/quickstart/advanced/quickstart-sdk.md:107]:
```suggestion
In order to activate the distributed validators, the cluster operator can retrieve the validators' associated deposit data from the lock file and use it to craft transactions to the `deposit()` method on the deposit contract.
```

File: docs/int/quickstart/advanced/quickstart-sdk.md [hunk 2]
```diff
@@ -88,21 +90,48 @@ Once the Obol-API returns a `configHash` string from the `createClusterDefinitio
...
+
+const depositContract = new ethers.Contract(
+  DEPOSIT_CONTRACT_ADDRESS, // 0x00000000219ab540356cBB839Cbe05303d7705Fa for Mainnet, 0xff50ed3d0ec03aC01D4C79aAd74928BFF48a7b2b for Goerli
+  depositContractABI, // https://etherscan.io/address/0x00000000219ab540356cBB839Cbe05303d7705Fa#code for Mainnet, and replace the address for Goerli
+  signer
+);
+
+const principal = BigNumber.from(validatorDepositData.amount); // 32 ethers
+
+const TX_VALUE = principal.mul(WEI_FACTOR);
```
Comment (OisinKyne) [inline docs/int/quickstart/advanced/quickstart-sdk.md:123]: Have you tested this? I would have thought principal coming from our API ends up being `"32000000000"`. So multiplying by a further 18 decimals leads to too many decimals and too high a value?

I would imagine we multiply by `10 ** 9` to get the outcome we want. 

Here's a calculator to help test https://eth-converter.com/

File: docs/int/quickstart/advanced/quickstart-sdk.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ import TabItem from '@theme/TabItem';
 The Obol-SDK is in an alpha state and should be used with caution.
```
Comment (OisinKyne) [inline docs/int/quickstart/advanced/quickstart-sdk.md:13]:
```suggestion
The Obol-SDK is in an alpha state and should be used with caution, particularly on mainnet. 
```

**Review by OisinKyne — COMMENTED** [review]
I'm not super confident this is right from eyeballing, we should test it, and then if we're happy, merge :) (If you've already tested it then I guess my bad 😅 )

File: docs/int/quickstart/advanced/quickstart-sdk.md (hunk 2 above)
Comment (HananINouman) [reply docs/int/quickstart/advanced/quickstart-sdk.md:123]: You are right. Fixing it

**Review by OisinKyne — APPROVED** [review]

**Pull request merged**